src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,8 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => (
+  <article className={`TodoInfo ${todo.completed && 'TodoInfo--completed'}`}>
```

Comment:
this trick will not work with classes `${todo.completed && 'TodoInfo--completed'}` - check your demo page and you will see `false` class many times
<img width="463" alt="image" src="https://github.com/mate-academy/react_static-list-of-todos-js/assets/6568745/a91e6c0a-6e11-404c-abee-269da90865f3">

So use classname library or use another approach